compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.td

```diff
@@ -496,4 +496,26 @@ def IREEGPU_BufferResourceCastOp : Op<IREEGPU_Dialect, "buffer_resource_cast", [
   let hasCanonicalizer = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// GlobalLoadDMAOp
+//===----------------------------------------------------------------------===//
+
+def IREEGPU_GlobalLoadDMAOp : Op<IREEGPU_Dialect, "global_load_dma", [
+    SameVariadicOperandSize]> {
+  let summary = "Does a global load DMA operation";
+  let description = [{
+    TODO: Add description
```

Comment:
Same here, would be good to expand on the semantics of the op before landing (especially given that they are non-obvious).

Reply:
updated